text/0026-rest-api/rest-api.yaml

```diff
@@ -0,0 +1,1777 @@
+        parent1MessageId:
+          type: string
+          description: The identifier of the first message this message references.
+        parent2MessageId:
+          type: string
+          description: The identifier of the second message this message references.
+        payload:
+          oneOf:
+            - $ref: '#/components/schemas/TransactionPayload'
+            - $ref: '#/components/schemas/MilestonePayload'
+            - $ref: '#/components/schemas/IndexationPayload'
+        nonce:
+          type: string
+          description: The nonce which lets this message fulfill the Proof-of-Work requirement.
+
+    TransactionPayload:
+      description: The Transaction Payload to be embedded into a message.
+      properties:
+        type:
+          type: integer
+          description: Set to value 0 to denote a Transaction Payload.
+        essence:
+          type: object
+          oneOf:
+            - $ref: '#/components/schemas/TransactionEssence'
+        unlockBlocks:
+          type: array
+          items:
+            oneOf:
+              - $ref: '#/components/schemas/SignatureUnlockBlock'
+              - $ref: '#/components/schemas/ReferenceUnlockBlock'
+
+    TransactionEssence:
+      description: Describes the essence data making up a transaction by defining its inputs and outputs and an optional payload.
+      properties:
+        type:
+          type: integer
+          description: Set to value 0 to denote a Transaction Essence.
+        inputs:
+          type: array
+          items:
+            oneOf:
+              - $ref: '#/components/schemas/UTXOInput'
+        outputs:
+          type: array
+          items:
+            oneOf:
+              - $ref: '#/components/schemas/SigLockedSingleOutput'
+        payload:
+          oneOf:
+            - $ref: '#/components/schemas/IndexationPayload'
+          nullable: true
+
+    UTXOInput:
+      description: Describes an input which references an unspent transaction output to consume.
+      properties:
+        type:
+          type: integer
+          description: Set to value 0 to denote an UTXO Input.
+        transactionId:
+          type: string
+          description: The BLAKE2b-256 hash of the transaction from which the UTXO comes from.
+        transactionOutputIndex:
+          type: integer
+          description: The index of the output on the referenced transaction to consume.
+    SigLockedSingleOutput:
+      description: Describes a deposit to a single address which is unlocked via a signature.
+      properties:
+        type:
+          type: integer
+          description: Set to value 0 to denote a SigLockedSingleOutput.
+        address:
+          anyOf:
+            - $ref: '#/components/schemas/Ed25519Address'
+        amount:
+          type: integer
+
```

Comment:
Need to add `SigLockedDustAllowanceOutput` type, and add as an alternative to `SigLockedSingleOutput`